python/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3
+ADD client.py /
+RUN pip install hazelcast-python-client
+CMD [ "python", "-u", "client.py" ]
```

Comment:
Minor: Mİssing new line at the end of the file

Reply:
updated